commands/bmc.nicole

```diff
@@ -73,11 +73,15 @@ function cmd_info_uptime {
 #   -b, --bmc   Show BMC version
 #   -o, --opfw  Show OpenPOWER Firmware version
 function cmd_info_version {
+  # If no options are given, show everything
+  [[ $# -eq 0 ]] && set -- -b -o
+
   local bmc_match='^BMC '
   local host_match='^Host '
   local no_match='no-match-pattern'
   local bmc=${no_match}
   local host=${no_match}
+
```

Comment:
This line does not look like someone improves the readable but it also does not look bad.
I think you should either remove it or add the same line into the `bmc.vegman`.

Reply:
Will do